backend/conftest.py

```diff
@@ -0,0 +1,66 @@
+from unittest.mock import AsyncMock
```

Comment:
This file has been moved inside the tests folder boss, there are fixtures for read and update you can use @mykie88

Reply:
omo the merge deleted it o, how come it is here again smh!

Reply:
thanks baba @man-of-mind for identifying this, all fixed now. Gracias!!!